src/main/java/org/victorrobotics/frc/dtlib/sensor/imu/DTIMUFaults.java

```diff
@@ -1,5 +0,0 @@
-package org.victorrobotics.frc.dtlib.sensor.imu;
-
-public interface DTIMUFaults<FAULT_TYPE> {
-    FAULT_TYPE internal();
-}
```

Comment:
Is there an enumeration of possible IMU faults we could define that would be the union of those supported by the two IMU models? Then returning one of those could be in the base interface.

Reply:
Actually, the NavX (an `AHRS` in code) doesn’t seem to support any faults whatsoever, ssh that’s why this was removed.

Reply:
[](https://www.kauailabs.com/public_files/navx-mxp/apidocs/java/com/kauailabs/navx/frc/AHRS.html)